.claude/agents/integration-qa.md

```
@@ -0,0 +1,153 @@
+  -d '{"message": "test"}' -o /dev/null -w "%{http_code}"
+
+# Fork (requires auth)
+curl -s -X POST http://localhost:8000/api/assistants/public/{id}/fork \
+  -H "Authorization: Bearer <token>" -o /dev/null -w "%{http_code}"
+```
+
+## Output Format
+
+For each issue found:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)